introduction/java_if_else/java_if_else.java

```diff
@@ -0,0 +1,27 @@
+import java.util.Scanner;
+
+public class Solution {
+  private static String isWeird(int numberToCheck) {
+    String answer = new String();
+    if ((numberToCheck % 2 == 1) && (numberToCheck <= 100) && (numberToCheck >= 1)) {
+      answer = "Weird"; 
+    } else {
```

Comment:
Can we put all `if` to the same level? Like

    if (..) {
      ...
    } else if(..) {
      ...
    } else if (..) {
      ...
    }